src/cryptography/hazmat/backends/openssl/x509.py

```diff
@@ -637,6 +639,267 @@ def _decode_inhibit_any_policy(backend, asn1_int):
     return x509.InhibitAnyPolicy(skip_certs)
 
 
+@utils.register_interface(x509.RevokedCertificate)
+class _RevokedCertificate(object):
+    def __init__(self, backend, x509_revoked):
+        self._backend = backend
+        self._x509_revoked = x509_revoked
+
+        self.__serial_number = None
```

Comment:
Double underscore is inconsistent with our typical approach of single underscore -- would you mind changing it?